stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -52,6 +54,9 @@ internal class PaymentSheetViewModel internal constructor(
         args.clientSecret
     )
 
+    private val mutableGooglePayCompletion = MutableLiveData<PaymentIntentResult>(null)
```

Comment:
i think you don't need the `null` initializer - if you just use an empty constructor i think when you .observe on it it won't fire , then you don't need the null check in the activity either, i think. we do this in the standalone app too